### PR TITLE
Add Sprint report - 32 (X-mas Sprint)

### PR DESCRIPTION
**Pull request by Ana06**
:christmas_tree:

### CODE REVIEW

File: _posts/development/2018-02-19-sprint-report-32.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+<img src="/images/posts/sprint_32-coolo_present.jpg" alt="Coolo's present" style="display:inline;width:40%">
+<img src="/images/posts/sprint_32-coolo-ruler.jpg" alt="Coolo's present" style="display:inline;width:40%">
+
+
+For the rest of users, that are still affected by this issue we have created an issue, which will be solved sooner rather than later:
+[#4525](https://github.com/openSUSE/open-build-service/issues/4525).
+
+
+# SUSE Studio Express
```
Comment (Ana06) [inline _posts/development/2018-02-19-sprint-report-32.md]: @hennevogel is ok to mention the Studio shutdown in the Sprint report? :relaxed:

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+<img src="/images/posts/sprint_32-coolo_present.jpg" alt="Coolo's present" style="display:inline;width:40%">
+<img src="/images/posts/sprint_32-coolo-ruler.jpg" alt="Coolo's present" style="display:inline;width:40%">
+
+
+For the rest of users, that are still affected by this issue we have created an issue, which will be solved sooner rather than later:
+[#4525](https://github.com/openSUSE/open-build-service/issues/4525).
+
+
+# SUSE Studio Express
```
Comment (Ana06) [inline _posts/development/2018-02-20-sprint-report-32.md]: @hennevogel is ok to mention the Studio shutdown in the Sprint report? :relaxed:

**Comment by DavidKang** [comment]
@Ana06, is the best sprint report of this year :clap: :clap: :clap:

**Comment by Ana06** [comment]
@DavidKang 

> @Ana06, is the best sprint report of this year :clap: :clap: :clap:

of the Chinese year? :unamused:

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+category: development
+author: "Ana María Martínez Gómez"
+---
+
+People of the Builds! And another Sprint is over and here is what the OBS frontend team has achieved in the last two weeks (2018-02-05 to 2018-02-16).
+
+# Ruby 2.5
+
+As every year, we got a new Ruby release for Christmas! :christmas_tree:
+So, even if it is always Christmas in the OBS team, Christmas is already over and it was already time to unwrap the present and move to Ruby 2.5.
```
Comment (bgeuken) [inline _posts/development/2018-02-20-sprint-report-32.md]: I think I would drop the second 'already'. It's a bit repetitive.

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+![OBS Christmas tree](/images/posts/sprint_32-xmas.jpg)
+
+
+# Releases
+
+## OBS 2.8 packages & images for openSUSE Leap 42.3
+
+Last sprint we worked on the new openSUSE Leap 42.3 images for OBS 2.8.
+Though didn't manage to release them because we didn't finish testing them.
```
Comment (bgeuken) [inline _posts/development/2018-02-20-sprint-report-32.md]: we didn't

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+We did that this sprint and [released](https://lists.opensuse.org/archive/opensuse-buildservice/2018-02/msg00009.html) the [images](https://download.opensuse.org/repositories/OBS:/Server:/2.8/images/) and [packages](https://download.opensuse.org/repositories/OBS:/Server:/2.8/openSUSE_42.3/x86_64/) last week. :smiley:
+
+
+# Cloud
+
+## Improve binary download page
+
+We have continued working on the cool cloud feature. :cloud:
```
Comment (bgeuken) [inline _posts/development/2018-02-20-sprint-report-32.md]: Nitpick, I would move the :cloud: before the dot. Or even drop the dot

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+<img src="/images/posts/sprint_32-coolo-ruler.jpg" alt="Coolo's present" style="display:inline;width:40%">
+
+
+For the rest of users, that are still affected by this issue we have created an issue, which will be solved sooner rather than later:
+[#4525](https://github.com/openSUSE/open-build-service/issues/4525).
+
+
+# SUSE Studio Express
+
+Last Wednesday, SUSE Studio was shutted down, and the Studios users are moving to OBS.
```
Comment (bgeuken) [inline _posts/development/2018-02-20-sprint-report-32.md]: was shut down

Comment (bgeuken) [reply _posts/development/2018-02-20-sprint-report-32.md]: and 'the Studio users'

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+Some of them are really excited about it:
+
+[![Happy Studio user Twit](/images/posts/sprint_32-studio-twet.jpg)](https://twitter.com/sshyukriev/status/963152868126707713)
+
+Learn more about SUSE Studio Express in our [new users welcoming blog post](http://openbuildservice.org/2017/09/27/suse-studio-express).
+
+
+## Tests for Relationship class
+
+As most of the Sprints, we have continued improving our Test Suite. :gem:
```
Comment (bgeuken) [inline _posts/development/2018-02-20-sprint-report-32.md]: Same here, drop the dot and remove the whitespace before the :gem:

**Comment by bgeuken** [comment]
The pictures of the ruler are a bit to big. Though I only checked in [GitHub](https://github.com/Ana06/obs-landing/blob/a0cc120b4d1fe7cb394f277f2a88fe902eefe328/_posts/development/2018-02-20-sprint-report-32.md)

File: _posts/development/2018-02-20-sprint-report-32.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: Highlights of the OBS frontend development - Sprint 32 (X-mas Sprint)
+category: development
+author: "Ana María Martínez Gómez"
```
Comment (bgeuken) [inline _posts/development/2018-02-20-sprint-report-32.md]: I am a bit split about this one. I know you've put quite some effort into this this time. On the other hand a couple of us wrote the different sections and we usually post them as OBS Team.

**Comment by Ana06** [comment]
@bgeuken 

> The pictures of the ruler are a bit to big. Though I only checked in GitHub

they are 40%:

```html
<img src="/images/posts/sprint_32-coolo-ruler.jpg" alt="Coolo's present" style="display:inline;width:40%">
```

**Comment by bgeuken** [comment]
Okay, then it probably is just a problem in markdown and no issue for us.

File: _posts/development/2018-02-20-sprint-report-32.md (hunk 5 above)
Comment (Ana06) [reply _posts/development/2018-02-20-sprint-report-32.md]: why? every sentence should end with a dot. And the :cloud: looks great everywhere. :rage:

**Pull request merged**